# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xemacs-beta/no-info.info.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- xemacs-21.5.27/man/Makefile.vanilla	2006-05-23 22:40:32.944231000 +0200
+++ xemacs-21.5.27/man/Makefile	2006-05-23 22:40:37.560519500 +0200
@@ -48,7 +48,6 @@
 	$(INFODIR)/custom.info \
 	$(INFODIR)/emodules.info \
 	$(INFODIR)/external-widget.info \
-	$(INFODIR)/info.info \
 	$(INFODIR)/lispref.info \
 	$(INFODIR)/internals.info \
 	$(INFODIR)/new-users-guide.info \
